modules/data.atmosphere/R/download.NOAA_GEFS_downscale.R

```diff
@@ -241,8 +242,14 @@ download.NOAA_GEFS_downscale <- function(outfolder, lat.in, lon.in, sitename, st
     dplyr::group_by_at(c("NOAA.member", "timestamp")) %>% 
     dplyr::summarize(surface_downwelling_shortwave_flux_in_air = mean(surface_downwelling_shortwave_flux_in_air))
   
+  ## Downscale Precipitation Flux 
```

Comment:
Would be good to have a tad more explanation about what's actually going on here.